jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/db2/DB2SqlGenerationVisitor.java

```diff
@@ -358,11 +365,11 @@ public String visit(SqlFunctionAggregateGroupConcat function) throws AdapterExce
         return builder.toString();
     }
 
-    
+
```

Comment:
```suggestion
```

I removed extra empty lines between methods. But in general in this file the indentation is also off.

Reply:
fixed